example/index.js

```diff
@@ -10,20 +10,18 @@ import {
   SolletExtensionWalletAdapter,
   SolletWalletAdapter,
   TorusWalletAdapter,
-} from '@solana/wallet-adapter-wallets';
+} from "@solana/wallet-adapter-wallets";
 
 const walletOptions = {
   wallets: [
     new PhantomWalletAdapter(),
     new SlopeWalletAdapter(),
-    new SolflareWalletAdapter({ network: 'devnet' }),
+    new SolflareWalletAdapter({ network: "devnet" }),
     new TorusWalletAdapter(),
-    new SolletWalletAdapter({ network: 'devnet' }),
-    new SolletExtensionWalletAdapter({ network: 'devnet' }),
+    new SolletWalletAdapter({ network: "devnet" }),
+    new SolletExtensionWalletAdapter({ network: "devnet" }),
```

Comment:
FWIW, in the Wallet Adapter examples, we have removed these, and now use just the `UnsafeBurnerWalletAdapter` for demonstration. The idea is that apps should be encouraged to pick some wallets rather than rely on a list of preferred defaults they just copy paste.